static/build/.tmp_update/runtime.sh

```diff
@@ -151,13 +151,13 @@ launch_main_ui() {
     mainUiBatPerc
 
     check_hide_recents
-    check_hide_expert
-
+    
     wifi_setting=$(/customer/app/jsonval wifi)
 
     start_audioserver
 
     # MainUI launch
+    mount_main_ui
```

Comment:
I think you should unmount it first. Maybe we should actually keep the expert/clean flag, and only remount if that changes (then move the first mount into main)